style.css

```diff
@@ -1,245 +1,328 @@
-  font-weight: 600;
-  font-size: 1.2em;
-  color: gray;
-  margin-top: 20px;
-}
+  .languages-title {
+    font-weight: 600;
+    font-size: 1.2em;
+    color: gray;
+    margin-top: 20px;
+  }
 
-.languages-head {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  width: 100%;
-  cursor: pointer;
-}
+  .languages-head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    cursor: pointer;
+  }
 
-.language-container {
-  display: flex;
-  flex-wrap: wrap;
-  gap: 12px;
-  margin-top: 20px;
-  margin-bottom: 36px;
-}
+  .language-container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 12px;
+    margin-top: 20px;
+    margin-bottom: 36px;
+  }
 
-.language-card {
-  padding: 20px;
-  width: 100px;
-  border-radius: 10px;
-  background: #f7f7f7;
-}
+  .language-card {
+    padding: 20px;
+    width: 100px;
+    border-radius: 10px;
+    background: #f7f7f7;
+  }
 
-.frameworks,
-.skills {
-  display: flex;
-  flex-wrap: wrap;
-  gap: 12px;
-  margin-top: 20px;
-  margin-bottom: 36px;
-}
+  .frameworks,
+  .skills {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 12px;
+    margin-top: 20px;
+    margin-bottom: 36px;
+  }
 
-.about-work-link {
-  color: #6071fe;
-  text-decoration: none;
-  margin-top: 20px;
-  font-size: medium;
-  border: #6071fe solid 1px;
-  border-radius: 10px;
-  padding: 10px;
-  width: 40%;
-}
+  .about-work-link {
+    color: #6071fe;
+    text-decoration: none;
+    margin-top: 20px;
+    font-size: medium;
+    border: #6071fe solid 1px;
+    border-radius: 10px;
+    padding: 10px;
+    width: 40%;
+  }
+
+  .about-work-link:hover {
+    background-color: #6071fe;
+    color: white;
+  }
+
+  .contact {
+    display: flex;
+    flex-direction: column;
+    background: #6071fe;
+    padding-top: 142px;
+    padding-bottom: 100px;
+    background-image: url("images/contact-form-bg.png");
+    background-size: cover;
+    border-radius: 100px 0 0;
+  }
 
-.about-work-link:hover {
-  background-color: #6071fe;
-  color: white;
+  .contact-title {
+    font-weight: 600;
+    text-align: center;
+    margin: 0;
+    margin-top: 10px;
+    font-size: 4em;
+    color: white;
+  }
+
+  .contact-subtitle {
+    font-weight: 100;
+    font-size: 1.5em;
+    text-align: center;
+    color: #d9dbff;
+    padding-left: 300px;
+    padding-right: 300px;
+    padding-bottom: 10px;
+  }
+
+  .form-section {
+    display: flex;
+    flex-direction: column;
+    border-radius: 0 50px 0 0;
+  }
+
+  input {
+    border-radius: 5px;
+    border: 1px solid #6071fe;
+    padding: 20px;
+    margin-left: 496px;
+    margin-right: 496px;
+    margin-bottom: 40px;
+  }
+
+  input:focus {
+    outline: none;
+  }
+
+  textarea {
+    height: 100px;
+    border-radius: 5px;
+    border: 1px solid #6071fe;
+    padding: 10px;
+    margin-left: 496px;
+    margin-right: 496px;
+    margin-bottom: 10px;
+    resize: none;
+  }
+
+  .submit {
```

Comment:
- [ ] [OPTIONAL] I Kindly suggest that you fix the submit button so that it is exactly like the Figma Design

| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="207" alt="image" src="https://user-images.githubusercontent.com/52934128/186465827-bacc4de3-bf66-4a69-9a67-1895b739959c.png">  | <img width="316" alt="image" src="https://user-images.githubusercontent.com/52934128/186465660-59116021-9da7-4d1e-94b7-a18bda9e836c.png">  |